Classes/Console/Command/InstallTool/UnlockInstallToolCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Helhum\Typo3Console\Command\InstallTool;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Command\RelatableCommandInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use TYPO3\CMS\Core\Core\Environment;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class UnlockInstallToolCommand extends Command implements RelatableCommandInterface
+{
+    public function getRelatedCommandNames(): array
+    {
+        return [
+            'typo3_console:installtool:lock',
+        ];
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Unlock Install Tool');
+        $this->setHelp('Allow install tool access.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $fileName = Environment::getLegacyConfigPath() . '/ENABLE_INSTALL_TOOL';
+
+        // The file should also be written if it already exists to get a new
+        // time stamp
+        GeneralUtility::writeFile($fileName, '');
+
+        if (!@is_file($fileName)) {
+            $output->writeln('<error>Could not create file \'typo3conf/ENABLE_INSTALL_TOOL\'.</error>');
+
+            return 1;
+        }
+
+        $output->writeln(
+            '<info>Install Tool has been unlocked and can be accessed now at \'typo3/install.php\'.</info>'
```

Comment:
Not sure if the url is valid for all TYPO3 versions like this. Should be taken into account for a potential back port...

Reply:
Yes, this is valid since TYPO3v8: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.0/Breaking-81951-InstallToolEntryPointRemoved.html:

> `typo3/install.php` is the new way to access the install tool, available since TYPO3 8 LTS.